test/jdk/tools/jar/ReleaseBeforeFiles.java

```diff
@@ -23,14 +23,12 @@
 
 /*
  * @test
- * @bug 8167237
+ * @bug 8167237 8316804
```

Comment:
Yes, looked at it and thought the this test's name fits better: it's about checking that `--release` appears before any `FILES` argument. Not?

Reply:
We missing tests to properly exercise this combination of options with MR JARs.  I pointed to multiRelease/Basic.java because it tests --describe-module with MR JARs. I think we need to add more tests there to ensure that it prints the correct description when a MR JAR has a module-info.class in a versioned section, or the module-info.class in a version section overrides the module-info.class in the base section. It might be we need further tests to checking ordering of options but we appear to missing some basic tests too.